app/services/api/v1x2/catalog/tagging_service.rb

```diff
@@ -43,13 +43,13 @@ def object
         end
 
         def call_remote_service(options = {:limit => QUERY_LIMIT})
-          REMOTE_SERVICES[@app_name].call do |api|
+          REMOTE_SERVICES[@app_name].first.call do |api|
             api.send(api_method_name, @object_id, options)
           end
         end
 
         def tags
-          api_tag_klass = "#{REMOTE_SERVICES[@app_name].name}ApiClient::Tag".classify.constantize
+          api_tag_klass = REMOTE_SERVICES[@app_name].last
```

Comment:
Previously type `::SourcesApiClient::Tag` will be resolved at runtime.  Since we never really test this case, no error was ever raised.
Now this type is resolved when the file is loaded and it fails immediately.

Sure we can change the structure to Hash for better reading.